sufia-models/app/actors/sufia/generic_file/actor.rb

```diff
@@ -105,7 +105,7 @@ def save_and_record_committer
     end
 
     def push_characterize_job
-      Sufia.queue.push(CharacterizeJob.new(@generic_file.id))
+      Sufia.queue.push(CharacterizeJob.new(generic_file.id))
```

Comment:
This seems unrelated to the declared purpose of the pull request. Can you separate this to a different PR?